packages/client/src/components/ViewDropDown/ViewDropdown.component.js

```diff
@@ -23,7 +27,8 @@ const ViewDropdown = ({ options, label, ...props }) => {
 };
 
 ViewDropdown.propTypes = {
-  options: PropTypes.string.isRequired,
+  // eslint-disable-next-line react/forbid-prop-types
```

Comment:
Please remove this and fix the prop types instead. You want to use `arrayOf(PropTypes.shape(...))`.

Reply:
@miltonhultgren we have tried with arrayOf(PropTypes.shape(...)) but we are getting error as we have both strings and numbers so we have changed it to .any . Please suggest us how to use that and change it to arrayOf(PropTypes.shape(...)). Thanks